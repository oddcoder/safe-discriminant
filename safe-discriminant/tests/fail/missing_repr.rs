use safe_discriminant::Discriminant;

#[derive(Discriminant)]
pub enum Foo {
    A = 0,
    B = 1,
}

fn main() {}
