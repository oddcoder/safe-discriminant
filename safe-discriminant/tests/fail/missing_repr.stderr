error: Discriminant requires a `#[repr(x)] where x is one of u8, i8, u16, i16, u32, i32, u64, i64.
 --> tests/fail/missing_repr.rs:4:10
  |
4 | pub enum Foo {
  |          ^^^
